examples/minwebgl/outline/src/main.rs

```diff
@@ -0,0 +1,417 @@
+//! Implementation of JFA outline
+
+use minwebgl as gl;
+use gl::
+{
+  GL,
+  web_sys::{
```

Comment:
Opening braces should start on the new line. Overall fix the formatting everywhere. In shaders as well